### PR TITLE
OSASINFRA-3151: CCPMSO jobs for OpenStack platform

### PR DESCRIPTION
**Pull request by EmilienM**
These are the CCPMSO jobs for OpenStack, including a weekly periodic job that will send us slack notifications when it fails.

### CODE REVIEW

**Comment by EmilienM** [comment]
/pj-rehearse ack

Jobs can't be run now, the code is WIP in CCPMSO.

**Comment by EmilienM** [comment]
/cc pierreprinetti 
just FYI about this PR

File: ci-operator/config/openshift/cluster-control-plane-machine-set-operator/openshift-cluster-control-plane-machine-set-operator-release-4.14__periodics.yaml [hunk 1]
```diff
@@ -97,6 +97,20 @@ tests:
           cpu: 100m
       timeout: 3h0m0s
     workflow: ipi-gcp
+- as: e2e-openstack
+  interval: 24h
```
Comment (pierreprinetti) [inline ci-operator/config/openshift/cluster-control-plane-machine-set-operator/openshift-cluster-control-plane-machine-set-operator-release-4.14__periodics.yaml:101]: Do we need it every day? Do you expect changes to come from code we're not covering with presubmits?

**Review by JoelSpeed — COMMENTED** [review]
No issues, just want to check the interval

File: ci-operator/config/openshift/cluster-control-plane-machine-set-operator/openshift-cluster-control-plane-machine-set-operator-release-4.14__periodics.yaml (hunk 1 above)
Comment (damdo) [reply ci-operator/config/openshift/cluster-control-plane-machine-set-operator/openshift-cluster-control-plane-machine-set-operator-release-4.14__periodics.yaml:101]: Correct. This was suggested from TRT.

Comment (EmilienM) [reply ci-operator/config/openshift/cluster-control-plane-machine-set-operator/openshift-cluster-control-plane-machine-set-operator-release-4.14__periodics.yaml:101]: yeah good idea. let's make it run every week?

Comment (EmilienM) [reply ci-operator/config/openshift/cluster-control-plane-machine-set-operator/openshift-cluster-control-plane-machine-set-operator-release-4.14__periodics.yaml]: I updated to weekly, let me know if that's ok.

Comment (EmilienM) [reply ci-operator/config/openshift/cluster-control-plane-machine-set-operator/openshift-cluster-control-plane-machine-set-operator-release-4.14__periodics.yaml]: sorry I didn't see Joel's comment but IMO daily is too much... and we don't have that much of capacity to run it.
I'll suggest 72h or 1/week. Are you ok with it?

Comment (damdo) [reply ci-operator/config/openshift/cluster-control-plane-machine-set-operator/openshift-cluster-control-plane-machine-set-operator-release-4.14__periodics.yaml]: What do you mean by capacity? IMO we should align it with the rest of the jobs which run daily to have a comparable signal. This is based on TRT's suggestion, which we followed.

**Review by damdo — COMMENTED** [review]
Aside from the periodic interval, LGTM

File: ci-operator/config/openshift/cluster-control-plane-machine-set-operator/openshift-cluster-control-plane-machine-set-operator-release-4.14__periodics.yaml (hunk 1 above)
Comment (EmilienM) [reply ci-operator/config/openshift/cluster-control-plane-machine-set-operator/openshift-cluster-control-plane-machine-set-operator-release-4.14__periodics.yaml]: by capacity I mean our OpenStack clouds can't afford a bunch of daily CI jobs but I'll stop arguing here and TRT's suggestion, we'll see how it works in the future.

**Review by damdo — COMMENTED** [review]
/lgtm

**Pull request merged**